<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="8" column="0">
     <widget class="QSlider" name="horizontalSlider_coff2">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSlider" name="horizontalSlider_A">
            <property name="minimum">
             <number>-15</number>
            </property>
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_A">
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSlider" name="horizontalSlider_B">
            <property name="minimum">
             <number>-15</number>
            </property>
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_B">
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSlider" name="horizontalSlider_C">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-15</number>
            </property>
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_C">
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSlider" name="horizontalSlider_D">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_D">
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QSlider" name="horizontalSlider_E">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_E">
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QSlider" name="horizontalSlider_F">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_F">
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Shot">
          <property name="text">
           <string>Shot</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QPlainTextEdit" name="plainTextEdit_Output">
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_Clear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="2" column="2">
            <widget class="QCheckBox" name="checkBox_invert">
             <property name="text">
              <string>Invert</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox_JoystickIntegrateAngle">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Joystick</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QCheckBox" name="checkBox_FuseBlown">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>FuseBlown</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="Line" name="line_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="Line" name="line_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_En24">
             <property name="text">
              <string>En_24</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="Line" name="line_8">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QCheckBox" name="checkBox_maxSpeed">
             <property name="text">
              <string>MaxSpeed</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QComboBox" name="comboBox_TimeShot">
             <item>
              <property name="text">
               <string>single</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>two</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>all</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="2">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QDial" name="dial_angle_roll">
               <property name="minimum">
                <number>-45000</number>
               </property>
               <property name="maximum">
                <number>45000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0" alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_roll">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <italic>false</italic>
                </font>
               </property>
               <property name="text">
                <string>Roll</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QDial" name="dial_angle_pitch">
               <property name="minimum">
                <number>-45000</number>
               </property>
               <property name="maximum">
                <number>45000</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0" alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_pitch">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Pitch</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0" alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="label_yaw">
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Yaw</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QDial" name="dial_angle_yaw">
               <property name="minimum">
                <number>-45000</number>
               </property>
               <property name="maximum">
                <number>45000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="Line" name="line_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QSlider" name="horizontalSlider_HighPosition">
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QSlider" name="horizontalSlider_coff1">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QComboBox" name="comboBox_SerialPorts"/>
          <widget class="QPushButton" name="pushButton_Connect">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_ColsetSerialPort">
           <property name="text">
            <string>Closet</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>  IP - addres :   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_IP">
          <property name="text">
           <string>192.168.1.104</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="pushButton_SendIP">
          <property name="text">
           <string>Send-IP</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_mode">
        <item>
         <property name="text">
          <string>--------mode--------</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Joystik-Controls</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Speed-Controls</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Angle-Controls</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>InvidualControls-Position</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>InvidualControls-Velocity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_Status">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="pushButton_JoystikOn">
          <property name="text">
           <string>-- Joystick OFF --</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_ShotTime">
          <item>
           <property name="text">
            <string>100</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Timer2">
          <property name="text">
           <string>Timer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="Line" name="line_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>High Postion</string>
      </property>
     </widget>
    </item>
   </layout>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>720</y>
      <width>176</width>
      <height>17</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1040</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
